bazel/repository_locations.bzl

```diff
@@ -1,15 +1,18 @@
 REPOSITORY_LOCATIONS = dict(
     envoy = dict(
-        # envoy fork starting from 1.25.4 with cherry-picks:
+        # envoy 1.25.6 with patches applied:
         # https://github.com/envoyproxy/envoy/pull/25856
```

Comment:
i think these comments can be removed IF we are no longer depending on envoy-fork

Reply:
[218a00a](https://github.com/solo-io/envoy-gloo/pull/235/commits/218a00a69d2a50773a14840705cc81201fe8baf5)